labs/cicd-artifact-store/README.md

```diff
@@ -0,0 +1,80 @@
+# CICD-artifact-store
+
+An easy to follow tutorial on Integrating Apigee into you Continuous Integration
+ and Continuous Deployment pipeline in GCP.
+
+**Note** This is a sample Apigee API project for use with GCP Cloud Solution
+ *Integrating Apigee into your CI/CD cycle*. The instructions below are a
+ simplified form of the tutorial covered in the solution.
+
+## Prerequisites
+
+ 1. Select an existing Apigee X or hybrid org. Signup for an Apigee trial
+  otherwise.
+ 2. In GCS, create a bucket in the same region as your Apigee org
+ 3. In IAM, grant following roles to default Cloud Build service account
+  *[project_number]@cloudbuild.gserviceaccount.com*
+       * Apigee API Admin
+       * Apigee Environment Admin
+       * Secret Manager Secret Accessor
+
+## Usage
+
+### Continuous Integration
+
+ 1. In Apigee console, create API product *MockProductDev* for API MockTarget
+  and path /*
+ 2. In Apigee console, create developer *mocktargetqa@example.com*
+ 3. In Apigee console, create developer app *MockTargetAppQA* for *MockProduct*
```

Comment:
It would be nice if this could be performed by Maven using the edge.json file that you already have, but I understand you need this as a prerequisite to populate Secret Manager.

Maybe just one to think about for a future improvement.

Reply:
Trading off production-worthy, fully-automated, parameterized code to easily understandable learning aid. Hoping not to automate this in the future. Doc points to intermediate, advanced content that cover this aspect.